frontend/src/component/tags/CreateTagType/CreateTagType.tsx

```diff
@@ -70,12 +73,15 @@ const CreateTagType = () => {
                 handleSubmit={handleSubmit}
                 handleCancel={handleCancel}
                 tagName={tagName}
-                setTagName={setTagName}
                 tagDesc={tagDesc}
+                color={color}
+                setTagName={setTagName}
                 setTagDesc={setTagDesc}
+                setColor={setColor}
                 mode='Create'
                 clearErrors={clearErrors}
                 validateNameUniqueness={validateNameUniqueness}
+                isTagTypeColorEnabled={isTagTypeColorEnabled}
```

Comment:
why is this component padding boolean it doesn't need? can the target component get it itself from a hook?

Reply:
Since I need the tag type flag in the shared useTagTypeForm component, and this hook essentially controls both the Create and Edit form, I thought it made sense to add it here and propagate it. I don't have strong feelings, I can add it directly in the form if you think that makes more sense.